client/src/app/app.component.spec.ts

```diff
@@ -20,10 +17,9 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'alta-front'`, () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.componentInstance;
-    expect(app.title).toEqual('alta-front');
-  });
-
+  // it(`should have as title 'alta-front'`, () => {
+  //   const fixture = TestBed.createComponent(AppComponent);
+  //   const app = fixture.componentInstance;
+  //   expect(app.title).toEqual('alta-front');
+  // });
```

Comment:
why are you commenting out test code here?

Reply:
I should probably delete it, it was @makchamp  test that is not any more valuable since I changed around the stuff.